draft-jholland-quic-multicast.md

```diff
@@ -611,6 +606,7 @@ Capabilities Flags is a bit field structured as follows:
 
```

Comment:
Is the field bid-endian or little-endian?

Reply:
For an 8-bit field there's no difference between big and little endian, is there?  But I suppose a bit diagram is appropriate here to show which bit position.

Reply:
ah I'll correct myself, QUIC is pretty clear in https://www.rfc-editor.org/rfc/rfc9000.html#section-1.3-5 on that.
But doing it with explicit field values as in the diagram works well too.